decidim-sortitions/app/models/decidim/sortitions/sortition.rb

```diff
@@ -13,9 +13,12 @@ class Sortition < ApplicationRecord
       include Decidim::Loggable
       include Decidim::Comments::Commentable
       include Decidim::Randomable
+      include Decidim::TranslatableResource
 
       component_manifest_name "sortitions"
 
+      translatable_fields :title, :selected_proposals, :witnesses, :additional_info, :cancel_reason, :candidate_proposals
```

Comment:
I think `selected_proposals` is not a translatable field, can you check?

Reply:
Fixed!